src/Console/InstallHostpinnaclePackage.php

```
@@ -5,42 +5,71 @@
-        $this->info('Installing Laravel Mpesa......');
+        $this->info('Installing Laravel Hostpinnacle......');
         $this->info('Publishing hostpinnacle configuration');
 
         if (!$this->configExists('hostpinnacle.php')) {
             $this->publishConfiguration();
             $this->info('Publishing hostpinnacle configuration');
         } else {
             if ($this->shouldOverwriteConfig()) {
-                $this->info('Overwitting hostpinnacle configuration file......');
+                $this->info('Overwriting hostpinnacle configuration file......');
                 $this->publishConfiguration($force = true);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$force'. (undefined)

(UnusedLocalVariable)